app/com/gu/floodgate/Application.scala

```diff
@@ -34,7 +34,7 @@ class Application(
     TextFormat.write004(writer, CollectorRegistry.defaultRegistry.metricFamilySamples())
     Result(
       ResponseHeader(200),
-      HttpEntity.Strict(ByteString(writer.toString), Some("text/plain; version=0.0.4; charset=UTF-8")))
+      HttpEntity.Strict(akka.util.ByteString(writer.toString), Some("text/plain; version=0.0.4; charset=UTF-8")))
```

Comment:
Is this right that we're specifically referencing akka here, or is it to do with the Play dependency (and therefore fair use)?  If that's the case, might be worth adding a comment to the code?

Reply:
Yes, we are explicitly referencing Akka. That's because we need to feed an Akka `ByteString` into Play!, not a Pekko `ByteString` (even though they are actually the same thing, the compiler differentiates them because they are not _necessarily_ the same thing).

I don't think that this needs an explicit comment in the code, because it's obvious that we are only converting into an Akka class at the last minute explicitly to feed Play (via `HttpEntity.Strict()`)

Reply:
TBH I think this was a fair question. Explaining the _why_ of something can be as important as the code self-describing the _what_, especially for anyone looking at it now, or in the future, who may not have the same familiarity as the author.